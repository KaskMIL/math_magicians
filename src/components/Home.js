import React from 'react';
import Header from './Header';
import styles from './styles/Home.module.css';

function Home() {
  return (
    <>
      <Header />
      <main className={styles.home}>
        <h1>Welcome to our page!</h1>
        <article>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Rem labore
            laboriosam, corporis ipsam, tenetur accusantium amet nemo architecto
            quas sed sint est cumque dicta itaque voluptatibus numquam mollitia
            vel assumenda.
          </p>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ducimus
            accusantium natus obcaecati rerum est error, incidunt dignissimos,
            quia ut perspiciatis animi eius delectus neque alias soluta quisquam
            aut culpa ratione.
          </p>
        </article>
      </main>
    </>
  );
}

export default Home;
