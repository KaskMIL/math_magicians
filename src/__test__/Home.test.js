import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Home from '../components/Home';

test('Render correctly', () => {
  const tree = renderer
    .create(
    <BrowserRouter>
      <Home />
    </BrowserRouter>
      )
    .toJSON();
  expect(tree).toMatchSnapshot();
});