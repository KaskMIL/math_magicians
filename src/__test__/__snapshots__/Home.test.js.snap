// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly 1`] = `
Array [
  <header
    className="header"
  >
    <h1>
      Magic Magicians
    </h1>
    <nav>
      <ul>
        <li>
          <a
            className="link"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            className="link"
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            className="link"
            href="/about"
            onClick={[Function]}
          >
            About
          </a>
        </li>
      </ul>
    </nav>
  </header>,
  <main
    className="home"
  >
    <h2>
      Welcome to our page!
    </h2>
    <article>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Rem labore laboriosam, corporis ipsam, tenetur accusantium amet nemo architecto quas sed sint est cumque dicta itaque voluptatibus numquam mollitia vel assumenda.
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Ducimus accusantium natus obcaecati rerum est error, incidunt dignissimos, quia ut perspiciatis animi eius delectus neque alias soluta quisquam aut culpa ratione.
      </p>
    </article>
  </main>,
]
`;
